addons/account/static/src/components/tests_shared_js_python/tests_shared_js_python.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+import { rpc } from "@web/core/network/rpc";
+import { registry } from "@web/core/registry";
+
+import {
+    adapt_price_unit_to_another_taxes,
+    eval_taxes_computation,
+    eval_taxes_computation_prepare_context,
+    prepare_taxes_computation,
+} from "@account/helpers/account_tax";
+
+import { xml, onWillStart, useState, Component } from "@odoo/owl";
+
+export class TestsSharedJsPython extends Component {
+    static template = xml`
+        <button t-attf-class="#{state.done ? 'text-success' : ''}" t-on-click="processTests">Test</button>
```

Comment:
Does `done` really mean that the test succeeded?
Or just that the test ran?

Reply:
@william-andre just that the test ran